src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/name-android-activity.mdx

```diff
@@ -59,12 +77,23 @@ For general info on using this API, see the [Android SDK API usage guide](/docs/
   </tbody>
 </table>
 
-## Examples
+## Examples [#examples]
 
 ### Set activity name
 
-Example of setting an activity name:
+Here's an example of setting the activity name:
 
-```
+### Java [#java]
+
+```java
 NewRelic.withLaunchActivityName($activityName)
 ```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
we need to change android example too.

Reply:
Java example:        NewRelic.withApplicationToken(apptoken).withLaunchActivityName().start(this.getApplication());

Reply:
```suggestion
```kotlin
   NewRelic.withApplicationToken("apptooken").withLaunchActivityName($activityName).start(applicationContext)
   
```
```